### PR TITLE
example: add angular architect example

### PR DESCRIPTION
**Pull request by jbedard**

### CODE REVIEW

**Comment by alexeagle** [comment]
FYI @jialipassion

I think this could use a lib in addition to an app, demonstrating that it's possible to do an Nx-granularity dependency graph.
Also would be great to have `ng test` working.

**Review by alexeagle — COMMENTED** [review]
Could we make a first commit that's just the output of `ng new`, then subsequent commit that adds Bazel? that way those migration commits can be useful as documentation

**Comment by alexeagle** [comment]
I'd just make this directory named "angular" - if/when we make some other example that has a more custom build system with sth like esbuild/ngtsc then I think that will be the disambiguated one

**Comment by jbedard** [comment]
I really think we should have an example like that. Using architect like this example really misses out on most of the performance benefits bazel can bring.

File: angular/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+copy_to_bin(
+    name = "angular",
+    srcs = ["angular.json"],
+)
+
+# NOTE: project dist directories are under the project dir unlike the Angular CLI default of the root dist folder
+jq(
+    name = "tsconfig",
+    srcs = ["tsconfig.json"],
+    filter = ".compilerOptions.paths |= map_values(map(gsub(\"^dist/(?<p>.+)$\"; \"projects/\"+.p+\"/dist\")))",
```
Comment (alexeagle) [inline angular/BUILD.bazel:28]: could be followup- maybe we want this to go in a macro as well, so the .bzl file has the bazel magic, and this BUILD file reads a bit more like an end user would be okay maintaining

File: angular/WORKSPACE.bazel [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    sha256 = "f74beaad49fb904c42b767b28179e5a67200a4bcfd032a2c80015add5bfe8b20",
+    strip_prefix = "bazel-lib-de6e9ef843358bbf1ae2420efb6cac1d05f86b30",
+    url = "https://github.com/aspect-build/bazel-lib/archive/de6e9ef843358bbf1ae2420efb6cac1d05f86b30.tar.gz",
+)
+
+load("@aspect_bazel_lib//lib:repositories.bzl", "aspect_bazel_lib_dependencies", "register_jq_toolchains")
+
+aspect_bazel_lib_dependencies()
+
+register_jq_toolchains(version = "1.6")
```
Comment (alexeagle) [inline angular/WORKSPACE.bazel:14]: nit: you can omit version here

**Review by alexeagle — APPROVED** [review]

**Pull request merged**